### PR TITLE
Fix glide event not firing

### LINKED ISSUE #1296: Glide event not firing
## EDDI version in which issue is found
3.4.0

## VoiceAttack version in which issue is found (as applicable)
1.7.5

## Steps to reproduce
Drop from supercruise/OC into glide, ((EDDI glide)) command is not being executed/event never being fired, maybe because the FSD doesn't change to "masslock" status unless dropping to 2000 metres or lower altitude.

## Expected
Plugin command : '((EDDI glide))' being executed in VA.

## Observed
Nothing happens when dropping into glide. FSD doesn't get mass locked early enough (anymore)?

## Investigation
Making sure near_surface should be true by asking for coordinates (lat/long, heading, i. e. {DEC:Status latitude} et cetera). And checking all glide status variables (EDDI glide body/bodytype/gliding/system), all values are Not Set.

Maybe it could check for the drop altitude (25,000 m?) and the static ship velocity when in glide of 2,500 m/s?

### PR DESCRIPTION
**Pull request by Tkael**
Revised glide event detection algorithm.
Revised `Entered normal space` event to adjust pauses during that event. 
Revised `Glide` event to adjust pauses during that event.
Strip "+", "++", and "+++" from station name suffixes with the {P()} function.
Fixes #1296

### CODE REVIEW

**Comment by richardbuckle** [comment]
Revised algorithm looks good.

**Comment by richardbuckle** [comment]
Fixes #1296

**Comment by richardbuckle** [comment]
Algorithm looks good. If you can assure me that the reduced pauses don't lead to EDDI talking over Air Traffic Control (a real pet peeve of mine) then we're good.

**Comment by Tkael** [comment]
Further tweaked scripts. Ready for review. :-)

File: SpeechService/Translations.cs [hunk 1]
```diff
@@ -474,6 +474,11 @@ public static string Station(string station)
             {
                 station = STATION_MODEL_FIXES[station];
             }
+            // Strip "+", "++", and "+++" from station name suffixes
+            if (station.EndsWith("+"))
```
Comment (richardbuckle) [inline SpeechService/Translations.cs]: `String.TrimEnd()` is your friend here.

**Pull request merged**